presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1063,4 +1069,9 @@ public static boolean isPushdownSubfieldsEnabled(Session session)
     {
         return session.getSystemProperty(PUSHDOWN_SUBFIELDS_ENABLED, Boolean.class);
     }
+
+    public static boolean isOptimizeFullOuterJoinWithCoalesce(Session session)
```

Comment:
Most likely when rebasing you have lost the actual usage of the session property. Please recover it before merging.